### PR TITLE
Concentration risk section, default scss

### PR DESCRIPTION
**Pull request by SaenkoM**
Concentration risk RPF section: https://trello.com/c/n3loMsLB/264-rpf-concentration-risk
Average Win/Loss RPF section: https://trello.com/c/31jJwTOY/265-rpf-average-win-loss
Logic and warning modal for RPF requests when sync is taking place: https://trello.com/c/k12lB6Gg/263-rpf-add-new-page-new-page-link-and-menu
Check triggers for `ledgers`, `wallets`, `average win/loss` and `concentration risk` sections

CRA default scss compilation

### CODE REVIEW

File: src/components/AverageWinLoss/AverageWinLoss.js [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+
+import { propTypes, defaultProps } from './AverageWinLoss.props'
+import parseChartData from './AverageWinLoss.helpers'
+
+const CURRENCIES = ['USD', 'JPY', 'EUR', 'GBP']
+
+class AverageWinLoss extends PureComponent {
+  constructor(props) {
+    super(props)
+    this.handleStartDateChange = this.handleDateChange.bind(this, 'start')
```
Comment (jarindr) [inline src/components/AverageWinLoss/AverageWinLoss.js]: any specific reason that you don't use arrow function to auto binds those fn ?

Comment (SaenkoM) [reply src/components/AverageWinLoss/AverageWinLoss.js]: that's the original style of the project, normally i use arrow functions

File: package.json [hunk 2]
```diff
@@ -64,10 +65,8 @@
   },
   "scripts": {
     "prepare": "node scripts/copyFiles",
-    "build-sass": "node-sass-chokidar --include-path ./src --include-path ./node_modules src/ -o src/",
-    "watch-sass": "npm run build-sass && node-sass-chokidar --include-path ./src --include-path ./node_modules src/ -o src/ --watch --recursive",
-    "start": "npm run prepare && npm run build-sass && react-scripts start",
-    "build": "npm run prepare && npm run build-sass && react-scripts build",
+    "start": "npm run prepare && react-scripts start",
```
Comment (jarindr) [inline package.json]: Are you sure the static .css file is not used anymore ?

File: src/components/AverageWinLoss/AverageWinLoss.js (hunk 1 above)
Comment (jarindr) [reply src/components/AverageWinLoss/AverageWinLoss.js]: I think let's just use arrow functions then.

File: package.json (hunk 2 above)
Comment (SaenkoM) [reply package.json]: yes, https://github.com/bitfinexcom/bfx-report-ui/pull/245/files#diff-1fdf421c05c1140f6d71444ea2b27638R19

File: src/components/AverageWinLoss/AverageWinLoss.js (hunk 1 above)
Comment (SaenkoM) [reply src/components/AverageWinLoss/AverageWinLoss.js]: @jarindr changed to arrow functions

File: src/components/Wallets/Wallets.js [hunk 3]
```diff
@@ -32,12 +32,13 @@ const {
 class Wallets extends PureComponent {
   constructor(props) {
     super(props)
+    const { currentTime } = props
     this.handleDateChange = this.handleDateChange.bind(this)
```
Comment (jarindr) [inline src/components/Wallets/Wallets.js]: one file missing here 😆

File: package.json (hunk 2 above)
Comment (jarindr) [reply package.json]: okay, just make sure the static files is not used for bundle injecting for dev mode. double check with someone that usually dev on this.

File: src/components/AverageWinLoss/AverageWinLoss.js [hunk 4]
```diff
@@ -0,0 +1,190 @@
...
+
+  handleTimeframeChange = (timeframe) => {
+    this.setState({ timeframe })
+  }
+
+  hasNewTime = () => {
+    const { params } = this.props
+    const { start: currStart, end: currEnd, timeframe: currTimeframe } = params
+    const { start, end, timeframe } = this.state
+    return (start ? start.getTime() !== currStart : !!start !== !!currStart)
```
Comment (jarindr) [inline src/components/AverageWinLoss/AverageWinLoss.js]: can you split those out a bit? eg
```
const isDiffStart = (start ? start.getTime() !== currStart : !!start !== !!currStart)
const isDiffEnd = (end ? end.getTime() !== currEnd : !!end !== !!currEnd)
const isDiffTimeframe = (timeframe !== currTimeframe)
return isDiffStart || isDiffEnd || isDiffTimeframe
```

File: src/components/Wallets/Wallets.js (hunk 3 above)
Comment (SaenkoM) [reply src/components/Wallets/Wallets.js]: @jarindr i was just slightly changing that component, of course i can refactor every bind in the project, there are a lot of them, but is it needed? :)

Comment (jarindr) [reply src/components/Wallets/Wallets.js]: not really :P

**Pull request merged**